app/models/customer.rb

```diff
@@ -1,4 +1,5 @@
 class Customer < ApplicationRecord
+  has_many :orders, dependent: :destroy
```

Comment:
The lesson wants you to remove the `dependent` option so that it's not so easy to wipe out a customer who still has orders.

```suggestion
  has_many :orders
```